### ISSUE TITLE
How to implement a global stop loss?

### ISSUE THREAD

**Issue by WinPoss** [freqtrade/freqtrade#9844]
## Describe your environment

  * Operating system: ubuntu 20.04.6
  * Python Version: 3.8.10
  * CCXT version: 4.2.25
  * Freqtrade Version: 2024.1
  
## Your question

*Ask the question you have not been able to find an answer in the [Documentation](https://www.freqtrade.io/en/latest/)*

Assuming I have 6 open positions, if the total loss of these 6 assets reaches 10% of the account's total capital, then sell all 6 assets. 

In other words, when the overall loss reaches 10% of the account's total capital, close all positions.

Thanks.

**Comment by xmatthias** [comment]
leaving aside the fact that i don't think that's a good idea (positions should be treated individually - a "global stop" will have odd side-effects i think) - freqtrade does not provide such functionality.

There's [guards](https://www.freqtrade.io/en/stable/plugins/#stoploss-guard) for [this](https://www.freqtrade.io/en/stable/plugins/#maxdrawdown) which will prevent new entries for some time - but i do think that existing trades should be "left alone" - you shouldn't exit all trades just because of other positions going wrong (this will lead to more losses, not fewer, which i assume is what you're expecting from such functionality).


Overall, you should be able to handle this in the `custom_exit()` function - though how exactly - i'll leave up to your imagination / creativity.
